packages/celotool/src/lib/genesis_constants.ts

```diff
@@ -19,6 +19,8 @@ export const TEMPLATE = {
     constantinopleBlock: 0,
     petersburgBlock: 0,
     istanbulBlock: 0,
+    churritoBlock: 0,
```

Comment:
this is used for every genesis block we generate. So for example whenever someone launches a new testnet.

For that i thin it should be configurable for such cases. Imagine you want to launch a blockchain where donut happens on block 1000.

Reply:
Good point.  I believe that should be easy enough to do.  I will add a commit.

Reply:
Updated the PR, so it no longer modifies TEMPLATE.